mercat/common/scenarios/unittest/node/simple_concurrent.yml

```diff
@@ -24,9 +24,9 @@ transactions:
   - sequence:
     - validate # validate the account creation phase
     - concurrent:
-      - issue Alice 50 ACME Mike approve
```

Comment:
It is odd that these and the other changes from my PR are showing up in the diff.
I think that you do not need to rebase to develop. Since none of my changes should confilct with yours.

Reply:
I have to rebase, because the circleci pulls the latest cryptography library since we call `git submodule update --init --remote`, and without your change it will fail to build, I had temporarily stopped it from doing `--remote` to get it to build.

Reply:
Sure, rebase if you have to. But either way, the PR diff should only show the changes of this PR ;)

Reply:
Yeah, blame github for that :D in fact all code review tools I've worked with have this problem, that's why I do my best not to have to rebase mid-PR. :)

Reply:
Sounds good, if it is githubu's problems. Then I am good with this PR.